vumi_http_retry/retries.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+
+def requests_key(prefix):
+    return ':'.join((prefix, 'requests'))
+
+
+def next_score(req):
+    dt = req['intervals'][req['attempts']]
+    return req['timestamp'] + dt
+
+
+def add_request(redis, prefix, req):
+    req = {
+        'timestamp': req['timestamp'],
+        'attempts': req.get('attempts', 0),
+        'request': req['request'],
+        'intervals': req['intervals']
+    }
```

Comment:
Just realised we aren't storing the owner id on requests. Wondering if that is okay. With the current plan, the owner id probably won't be needed here, but maybe there are reasons I'm not thinking of that would make it necessary to store it. Thoughts?